lib/rollbar/middleware/js.rb

```diff
@@ -133,6 +133,7 @@ def html_safe_if_needed(string)
       def append_nonce?
         defined?(::SecureHeaders) && ::SecureHeaders.respond_to?(:content_security_policy_script_nonce) &&
           defined?(::SecureHeaders::Configuration) &&
+          !::SecureHeaders::Configuration.get.csp.opt_out? &&
```

Comment:
To be fair, we already do not support 2.x because of the next line we are already using `Configuration.get`